### PR TITLE
Fixed milestone

### PR DESCRIPTION
**Pull request by christopher77**
add the milestone set by the user

add method /save_weight_wanted in routes.rb

### CODE REVIEW

File: routes.rb [hunk 1]
```diff
@@ -71,4 +62,31 @@
   end
 end
 
+###----christoph
+get '/milestone' do
+  if session[:user_email]
+    @current_user = User.find(session[:user_email])
+    measure_last = @current_user.measures.last
+    @ideal_weight = measure_last.calc_ideal_weight(@current_user.gender)
+    puts "----------------------------------"
```
Comment (condef5) [inline routes.rb]: You can removed the `puts` message you don't use it.

File: views/milestone.erb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+<form action="/save_weight_wanted" method="POST">
+  <label class="label">Enter the Weight you want to reach:</label>
+
+  <input
+    class="input is-success"
+    style="width:300px"
+    type="number"
+    placeholder="Whatever you want!"
+    name="weight_wanted"
+  />
```
Comment (diegotc86) [inline views/milestone.erb]: I suggest you to add a default value on the input. In that way the user would be aware of his last goal.

Comment (christopher77) [reply views/milestone.erb]: yes, you're right ,I gonna do it in the next commit , thanks

File: routes.rb (hunk 1 above)
Comment (christopher77) [reply routes.rb]: sorry, I forgot that I left this commands, I will correct them in the next commit, thank you

File: views/milestone.erb (hunk 2 above)
Comment (diegotc86) [reply views/milestone.erb]: Hi, the property "value" is empty.

Comment (christopher77) [reply views/milestone.erb]: Sorry, my fault, I will complete it in the next commit, thank you

**Pull request merged**